Chandra/cmd_states/cmd_states.py

```diff
@@ -393,14 +399,16 @@ def get_state0(date=None, db=None, date_margin=10, datepar='datestop'):
         db = Ska.DBI.DBI(dbi='sybase', server='sybase', user='aca_read',
                          database='aca')
 
-    # Date for which cmd_states are certainly reliable
-    definitive_date = DateTime(time.time() - date_margin * 86400.,
-                               format='unix').date
-    if date is None or date > definitive_date:
-        date = definitive_date
-    else:
+    if date is not None:
```

Comment:
But then you have made a subtle change in the logic and API.  The current API is that if `date is None` then use `definitive_date`, whatever that might be.  So if you put in a negative `date_margin` your new API gives a different result.  I was being very careful to maintain exactly the current logic / API.